alvr/server_openvr/cpp/alvr_server/alvr_server.cpp

```diff
@@ -474,9 +474,9 @@ void RegisterButton(void* instancePtr, unsigned long long buttonID) {
     ((Controller*)instancePtr)->RegisterButton(buttonID);
 }
 
-void SetViewsConfig(FfiViewsConfig config) {
+void SetViewParams(const FfiViewParams params[2]) {
     if (g_driver_provider.hmd) {
-        g_driver_provider.hmd->SetViewsConfig(config);
+        g_driver_provider.hmd->SetViewParams(params);
     }
 }
```

Comment:
This is passing around a gigantic struct, should rather be a reference

Reply:
But also it doesn't actually matter, this occurs in a couple places, actually fixing it is probably not worth it because berf will not be measurably impacted anyway.